sql/src/main/java/org/apache/druid/sql/calcite/rel/DruidQuery.java

```diff
@@ -682,7 +682,7 @@ private static RowSignature computeOutputRowSignature(
     }
   }
 
-  private VirtualColumns getVirtualColumns(final boolean includeDimensions)
```

Comment:
I was curious why this is neccessary - but the removal of `private` is not needed